metrics/core.go

```diff
@@ -1,119 +1,74 @@
 package metrics
 
 import (
-	"fmt"
-	"time"
-
-	"github.com/Sirupsen/logrus"
 	"github.com/prebid/prebid-cache/config"
-	"github.com/rcrowley/go-metrics"
-	"github.com/vrischmann/go-metrics-influxdb"
+	influx "github.com/prebid/prebid-cache/metrics/influx"
+	prometheus "github.com/prebid/prebid-cache/metrics/prometheus"
+	"time"
 )
 
-type MetricsEntry struct {
-	Duration   metrics.Timer
-	Errors     metrics.Meter
-	BadRequest metrics.Meter
-	Request    metrics.Meter
-}
-
-type MetricsEntryByFormat struct {
-	Duration       metrics.Timer
-	Errors         metrics.Meter
-	BadRequest     metrics.Meter
-	JsonRequest    metrics.Meter
-	XmlRequest     metrics.Meter
-	DefinesTTL     metrics.Meter
-	InvalidRequest metrics.Meter
-	RequestLength  metrics.Histogram
+/* Object to access metric engines     */
```

Comment:
Please use standard Go convention for comments:

```
Metrics represents zero or more MetricEngines.
```

Reply:
Done